telco-core/configuration/reference-crs-kube-compare/optional/logging/ClusterLogSubscription.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging
+spec:
+  channel: "stable"
+  name: cluster-logging
+  source: {{ .spec.source }}
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Automatic
```

Comment:
I know it's in the RDS with installPlanApproval: Automatic, but I'm pretty sure a lot of customers will actually set this to Manual and select a channel + provide a startingCSV

Reply:
I also think the mainline use case will be Manual. In the policy generation PR #20 I override to Manual. We can easily shift that to be in the underlying base CR as a follow on.

Reply:
@imiller0 So should we accept Manual for the is then? I'm unsure.